src/components/Tables/ProductsTable/ProductsTable.js

```diff
@@ -47,13 +47,28 @@ const ProductsTable = () => {
 
   let result = products;
   if (searchTerm) {
-    result = products.filter((product) =>
-      product.description.toLowerCase().includes(searchTerm.toLowerCase())
-    );
+    result = products.filter((product) => {
+      var digits = product._id.toString();
+      if (digits.includes(searchTerm)) return product;
+      else if (products.upc !== undefined) {
```

Comment:
sorry lilian! the most important fields for the search is yung product code, description, UPC and supplier code talaga eh. I think I mentioned it na rin previously basta andun yung UPC and supplier code din

Reply:
Added but cannot finalize logic since I don't know what is the final data type for supplier code. Meaning if it's just numbers or just strings, it has different ways.

Reply:
UPC and Supplier code are both strings so the logic is correct!